### DISCUSSION TITLE
localhost App.py can not open

### DISCUSSION THREAD

**Discussion by vincentvaldinata** [teamalgoritma/community#378] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76637079/193433262-d2d2fc5c-18e5-4bae-b864-8ad9f36864c1.png)
![image](https://user-images.githubusercontent.com/76637079/193433269-73cedd2c-6beb-4cd8-9451-0e17b9d29c62.png)
![image](https://user-images.githubusercontent.com/76637079/193433282-51abf758-0864-4b95-aea0-ded7718a715a.png)
localhostnya ga muncul, harusnya muncul page baru dengan tulisan botisrunning

**Comment by yosiaazarya** [comment, marked as answer]
Halo pak Vincent, script `app.py` digunakan untuk deployment ke heroku, bukan untuk dijalankan secara lokal. Adapun untuk mengecek secara loca dapat melakukan bot polling dengan menjalankan `bot_local.py`.

Keterangan "bot is running" akan muncul jika setelah bapak deploy ke heroku, lalu bapak buka link yang tersedia di heroku setelah success dalam build. Untuk mengecek apakah deployment sudah berjalan dengan baik, setelah deployment sukses di heroku, maka bapak dapat mengecek secara langsung ke telegram bot yang bapak buat, terima kasih.
